### PR TITLE
Apply translate to email message

### LINKED ISSUE #455: Apply translate to email message

### PR DESCRIPTION
**Pull request by AngeloCG97**
## Apply translate to email message

closes #455 

### What does this PR do?

- Apply translate email message 
  - Modify Actions 
  - Modify validations files
  - Modify queries
  - Modify funtions to send emails

### How should this be tested?

- Run make run
- Go to localhost:3000
- Go to infopage
- Go to profile page lifebank and sponsor

### CODE REVIEW

File: hapi/src/api/account.api.js [hunk 1]
```diff
@@ -67,9 +67,15 @@ const create = async ({ role, email, name, secret }) => {
...
 
   await historyApi.insert(transaction)
-
   try {
-    mailApi.sendVerificationCode(email, verification_code)
+    mailApi.sendVerificationCode(
+      email,
+      verification_code,
+      emailContent.subject,
+      emailContent.titule,
```
Comment (JustinCast) [inline hapi/src/api/account.api.js:75]: titule or title?

File: hapi/src/routes/change-password/change-password.handler.js [hunk 2]
```diff
@@ -33,9 +33,9 @@ module.exports = async ({ payload: { input } }) => {
       if (user)
         await mailUtils.sendConfirmMessage(
           input.email,
-          'Lifebank Change Password',
-          'Change Password',
-          'Your password has been successfully changed, thank you for donating life'
+          input.emailContent.subject,
+          input.emailContent.titule,
```
Comment (JustinCast) [inline hapi/src/routes/change-password/change-password.handler.js:37]: titule?

File: hapi/src/routes/credentials-recovery/credentials-recovery.handler.js [hunk 3]
```diff
@@ -26,7 +26,12 @@ module.exports = async ({ payload: { input } }) => {
       await mailUtils.sendCredentialsRecovery(
         input.email,
         user.update_user.returning[0].account,
-        tempSecret
+        tempSecret,
+        input.emailContent.subject,
+        input.emailContent.titule,
```
Comment (JustinCast) [inline hapi/src/routes/credentials-recovery/credentials-recovery.handler.js:31]: check word "titule"

File: hapi/src/utils/mail.js [hunk 4]
```diff
@@ -19,12 +26,12 @@ const sendVerificationCode = async (to, verficationCode) => {
...
     from,
     to,
-    subject: 'Lifebank Verfication Code',
+    subject,
     html: `
           <head>
             <meta charset="utf-8">
             <meta http-equiv="x-ua-compatible" content="ie=edge">
-            <title>Email verification</title>
+            <title>${titule}</title>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:34]: check this

File: hapi/src/utils/mail.js [hunk 5]
```diff
@@ -138,27 +145,27 @@ const sendVerificationCode = async (to, verficationCode) => {
                   <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
                     <tr>
                       <td align="left" bgcolor="#ffffff" style="padding: 36px 24px 0; font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif; border-top: 3px solid #f6f6f6;">
-                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;">Email verification</h1>
+                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;">${titule}</h1>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:148]: Check this

File: hapi/src/utils/mail.js [hunk 6]
```diff
@@ -380,7 +387,16 @@ const sendRegistrationRequest = async (to, lifebankData) => {
   })
 }
 
-const sendCredentialsRecovery = async (to, account, tempSecret) => {
+const sendCredentialsRecovery = async (
+  to,
+  account,
+  tempSecret,
+  subject,
+  titule,
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:395]: Check this

File: hapi/src/utils/mail.js [hunk 7]
```diff
@@ -397,12 +413,12 @@ const sendCredentialsRecovery = async (to, account, tempSecret) => {
...
     from,
     to,
-    subject: 'Lifebank Credentials Recovery',
+    subject,
     html: `
           <head>
             <meta charset="utf-8">
             <meta http-equiv="x-ua-compatible" content="ie=edge">
-            <title>Credentials Recovery</title>
+            <title${titule}</title>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:421]: check this

File: hapi/src/utils/mail.js [hunk 8]
```diff
@@ -515,17 +531,17 @@ const sendCredentialsRecovery = async (to, account, tempSecret) => {
                   <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600 px;">
                     <tr>
                       <td align="left" bgcolor="#ffffff" style="padding: 36px 24px 0; font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif; border-top: 3px solid #f6f6f6;">
-                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;">Credentials Recovery</h1>
+                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;"${titule}</h1>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:534]: check this

File: hapi/src/utils/mail.js [hunk 9]
```diff
@@ -546,7 +562,7 @@ const sendCredentialsRecovery = async (to, account, tempSecret) => {
   })
 }
 
-const sendConfirmMessage = async (to, subject, titulo, message) => {
+const sendConfirmMessage = async (to, subject, titule, message) => {
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:565]: check this

File: hapi/src/utils/mail.js [hunk 10]
```diff
@@ -568,7 +584,7 @@ const sendConfirmMessage = async (to, subject, titulo, message) => {
           <head>
             <meta charset="utf-8">
             <meta http-equiv="x-ua-compatible" content="ie=edge">
-            <title>${titulo}</title>
+            <title>${titule}</title>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:587]: check this

File: hapi/src/utils/mail.js [hunk 11]
```diff
@@ -681,7 +697,7 @@ const sendConfirmMessage = async (to, subject, titulo, message) => {
                   <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600 px;">
                     <tr>
                       <td align="left" bgcolor="#ffffff" style="padding: 36px 24px 0; font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif; border-top: 3px solid #f6f6f6;">
-                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;"> ${titulo}</h1>
+                        <h1 style="margin: 0; font-size: 32px; font-weight: bold; line-height: 48px; color: #BA0D0D;"> ${titule}</h1>
```
Comment (JustinCast) [inline hapi/src/utils/mail.js:700]: check this

File: webapp/src/components/CredentialsRecovery/CredentialsRecovery.js [hunk 12]
```diff
@@ -145,7 +152,12 @@ const CredentialsRecovery = ({ overrideBoxClass, overrideLabelClass }) => {
     setErrorMessage(null)
     changePassword({
       variables: {
-        ...user
+        ...user,
+        emailContent: {
+          subject: t('emailMessage.subjectChangePassword'),
+          titule: t('emailMessage.tituleChangePassword'),
```
Comment (JustinCast) [inline webapp/src/components/CredentialsRecovery/CredentialsRecovery.js:158]: check this word

File: webapp/src/components/Signup/Signup.js [hunk 13]
```diff
@@ -250,6 +250,12 @@ const Signup = ({ isHome, isModal, isSideBar }) => {
           variables: {
             role,
             email,
+            emailContent: {
+              subject: t('emailMessage.subjectVerificationCode'),
+              titule: t('emailMessage.tituleVerificationCode'),
```
Comment (JustinCast) [inline webapp/src/components/Signup/Signup.js:255]: check this

File: webapp/src/components/Signup/Signup.js [hunk 14]
```diff
@@ -311,6 +317,12 @@ const Signup = ({ isHome, isModal, isSideBar }) => {
         preRegisterLifebank({
           variables: {
             email,
+            emailContent: {
+              subject: t('emailMessage.subjectVerificationCode'),
+              titule: t('emailMessage.tituleVerificationCode'),
```
Comment (JustinCast) [inline webapp/src/components/Signup/Signup.js:322]: check this

File: webapp/src/language/en.json [hunk 15]
```diff
@@ -335,7 +335,14 @@
       "messageAproveAccount": "Your application for registration in Lifebank has been approved, thank you for being part of this beautiful project",
       "subjectChangePassword": "Lifebank Change Password",
       "tituleChangePassword": "Change Password",
-      "messageChangePassword": "Your password has been successfully changed, thank you for donating life"
+      "messageChangePassword": "Your password has been successfully changed, thank you for donating life",
+      "subjectCredentialsRecovery": "Lifebank Credentials Recovery",
+      "tituleCredentialsRecovery": "Credentials Recovery",
+      "messageCredentialsRecovery": "A new password has been generated for your account, with this you can now access Lifebank and continue donating life.",
+      "subjectVerificationCode": "Lifebank Email Verification",
+      "tituleVerificationCode": "Email Verification",
```
Comment (JustinCast) [inline webapp/src/language/en.json:343]: check this

File: webapp/src/language/es.json [hunk 16]
```diff
@@ -332,10 +332,17 @@
...
       "subjectAproveAccount": "Lifebank Cuenta Aprobada",
       "tituleAproveAccount": "Cuenta Aprobada",
-      "messageAproveAccount": "Su solicitud de registro en Lifebank ha sido aprobada, gracias por ser parte de este hermoso proyecto",
+      "messageAproveAccount": "Su solicitud de registro en Lifebank ha sido aprobada, gracias por ser parte de este hermoso proyecto.",
       "subjectChangePassword": "Lifebank Cambio de Contraseña",
       "tituleChangePassword": "Cambio de Contraseña",
-      "messageChangePassword": "Su contraseña ha sido cambiada con éxito, gracias por donar vida"
+      "messageChangePassword": "Su contraseña ha sido cambiada con éxito, gracias por donar vida.",
+      "subjectCredentialsRecovery": "Lifebank Recuperación de Credenciales",
+      "tituleCredentialsRecovery": "Recuperación de Credenciales",
```
Comment (JustinCast) [inline webapp/src/language/es.json:340]: check this

File: webapp/src/language/es.json [hunk 17]
```diff
@@ -332,10 +332,17 @@
...
+      "messageCredentialsRecovery": "Se ha generado una nueva contraseña para su cuenta, con esta ahora podes acceder a Lifebank y seguir donando vida.",
+      "subjectVerificationCode": "Lifebank Verificación del Email",
+      "tituleVerificationCode": "Verificación del Email",
```
Comment (JustinCast) [inline webapp/src/language/es.json:343]: check this

File: webapp/src/routes/RegisterLifebank/RegisterLifebank.js [hunk 18]
```diff
@@ -92,6 +92,11 @@ const RegisterLifebank = (props) => {
       createAccountLifebank({
         variables: {
           email,
+          emailContent: {
+            subject: t('emailMessage.subjectAproveAccount'),
+            titule: t('emailMessage.tituleAproveAccount'),
```
Comment (JustinCast) [inline webapp/src/routes/RegisterLifebank/RegisterLifebank.js:97]: check this

**Review by JustinCast — CHANGES_REQUESTED** [review]

File: webapp/src/routes/EditProfile/EditProfileBank.js [hunk 19]
```diff
@@ -470,9 +470,15 @@ const EditProfileBank = ({ profile, isCompleting, onSubmit, setField, loading, u
             showSchedule
           />
         </Box>
+<<<<<<< HEAD
+        <Box style={{ display: isCompleting && JSON.parse(profile.photos).length > 0 ? 'none' : '' }} className={classes.marginTitle}>
+          <Typography variant="h4">{t('profile.images')}</Typography>
+          <Typography variant="body1" />
+=======
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileBank.js:477]: Hey @AngeloCG97 watch this

File: webapp/src/routes/EditProfile/EditProfileBank.js [hunk 20]
```diff
@@ -530,6 +536,22 @@ const EditProfileBank = ({ profile, isCompleting, onSubmit, setField, loading, u
             </Box>
           )}
         </div>
+<<<<<<< HEAD
```
Comment (JustinCast) [inline webapp/src/routes/EditProfile/EditProfileBank.js:539]: Please check this conflicts

**Review by JustinCast — APPROVED** [review]

**Review by JustinCast — CHANGES_REQUESTED** [review]
Check the comments

File: webapp/src/routes/RegisterLifebank/RegisterLifebank.js [hunk 21]
```diff
@@ -92,6 +92,11 @@ const RegisterLifebank = (props) => {
       createAccountLifebank({
         variables: {
           email,
+          emailContent: {
+            subject: t('emailMessage.subjectAproveAccount'),
+            title: t('emailMessage.titleAproveAccount'),
```
Comment (JustinCast) [inline webapp/src/routes/RegisterLifebank/RegisterLifebank.js:97]: "approve" goes with a double v

File: webapp/src/routes/RegisterLifebank/RegisterLifebank.js [hunk 22]
```diff
@@ -92,6 +92,11 @@ const RegisterLifebank = (props) => {
...
+            message: t('emailMessage.messageAproveAccount'),
```
Comment (JustinCast) [inline webapp/src/routes/RegisterLifebank/RegisterLifebank.js:98]: "approve" goes with a double v

File: webapp/src/routes/RegisterLifebank/RegisterLifebank.js [hunk 23]
```diff
@@ -92,6 +92,11 @@ const RegisterLifebank = (props) => {
       createAccountLifebank({
         variables: {
           email,
+          emailContent: {
+            subject: t('emailMessage.subjectAproveAccount'),
```
Comment (JustinCast) [inline webapp/src/routes/RegisterLifebank/RegisterLifebank.js:96]: "approve" goes with a double v

File: webapp/src/language/en.json [hunk 24]
```diff
@@ -331,11 +331,18 @@
     },
     "emailMessage": {
       "subjectAproveAccount": "Lifebank Approve Account",
-      "tituleAproveAccount": "Approve Account",
+      "titleAproveAccount": "Approve Account",
```
Comment (JustinCast) [inline webapp/src/language/en.json:334]: "approve" goes with a double v

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by JustinCast — APPROVED** [review]

**Review by JeanVegaD — APPROVED** [review]

**Pull request merged**